### DISCUSSION TITLE
[Bug] Unable to mark notification as done

### DISCUSSION THREAD

**Discussion by connor4312** [community/community#2845] [Discussions] [answered]
I have two notifications that appear on https://github.com/notifications that I can't mark as done or remove

1. Clicking on the notification and going to the issue does not show the blue bar at the top
1. Selecting them in the notifications view and clicking "done" results in:

    - A request to POST https://github.com/notifications/beta/archive which returns an empty 401
    - A subsequent pair of GET requests which once again return the notifications

1. Selecting them in the notifications view and clicking "unsubscribe" results in:

    - A request to POST https://github.com/notifications/beta/unsubscribe which returns an empty 401
    - A subsequent pair of GET requests which once again return the notifications

This is very annoying since I use notifications as my "to do" inbox, and now there are two todos that I can never check off/remove.

If it's helpful, the full urls for the notification links are:

```
https://github.com/mixer/retrieval/pull/4?notification_referrer_id=MDE4Ok5vdGlmaWNhdGlvblRocmVhZDEzNTk4ODE4MzA6MjIzMDk4NQ%3D%3D
https://github.com/Azure/AKS/issues/1370?notification_referrer_id=MDE4Ok5vdGlmaWNhdGlvblRocmVhZDY3Nzc0NTY1NDoyMjMwOTg1#event-4059302072
```

**Comment by becca** [comment, marked as answer]
This area is currently designated for feedback for Discussions and GitHub for mobile. https://github.com/github/feedback/discussions/1

This is currently an issue the Notifications team is aware of and tracking a fix but I do not have an ETA for you on when a fix will land.

**Reply by tkapin** [reply]
Hi @becca, this discussion is almost two years old so could you please share an update on this? The API to mark notification as done is not listed at https://docs.github.com/en/rest/activity/notifications#mark-a-thread-as-read so I'm wondering what is the right way to do this. Thanks in advance!
